docs/sdk/python.md

```diff
@@ -158,9 +158,8 @@ All stateful functions may contain state by merely storing values within the ``c
 The data is always scoped to a specific function type and identifier.
 State values could be absent, ``None``, or a ``google.protobuf.Any``.
 
-<div class="alert alert-info">
```

Comment:
Note: I removed these tags because they were causing the link for `Remote modules` to not be rendering correctly.